project-management/src/main/java/life/qbic/projectmanagement/application/measurement/validation/ValidationService.java

```diff
@@ -23,20 +23,20 @@ public class ValidationService {
 
   private final NGSValidator ngsValidator;
 
-  private final ProteomicsValidator pxValidator;
+  private final ProteomicsValidator pxpValidator;
```

Comment:
why pxp?

Reply:
Just for consistency reasons, we call registerPxP in the measurmentservice and check for "pxp" properties in the validator